service-base/src/main/java/org/eclipse/hono/service/tenant/BaseTenantService.java

```diff
@@ -194,61 +193,76 @@ private void reply(final Message<JsonObject> request, final AsyncResult<TenantRe
      * Sends a response to a tenant request over the Vertx event bus.
      *
      * @param request The message to respond to.
-     * @param result The tenant result that should be conveyed in the response.
+     * @param tenantResult The tenant result that should be conveyed in the response.
+     * @throws NullPointerException If request is null.
```

Comment:
what happens if tenant result is `null`?

Reply:
I will add it to the causes for a NullPointerException.